### PR TITLE
fix(cornerstone-dicom-sr): Freehand SR hydration support

### PR DESCRIPTION
**Pull request by igoroctaviano**
- Related PR: https://github.com/OHIF/Viewers/pull/3996
- Optional: https://github.com/dcmjs-org/dcmjs/pull/386

### Context
The Freehand ROI annotation SR hydration was not working for both SR and Cornerstone viewports

### Changes & Results

- Freehand ROI SRs are created and hydrated properly in OHIF

### Testing

- Link to OHIF locally and create/load a Freehand ROI SR

#### PR

- [x] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

N/A

#### Public Documentation Updates

N/A

#### Tested Environment

- [x] "OS: 14.4.1 (23E224) 
- [x] "Node version: 16.14 
- [x] "Browser: Chrome Version 123.0.6312.59 (Official Build) (x86_64)
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: packages/tools/src/tools/annotation/PlanarFreehandROITool.ts [hunk 1]
```diff
@@ -243,7 +268,7 @@ class PlanarFreehandROITool extends ContourSegmentationBaseTool {
            */
           epsilon: 0.1,
         },
-        calculateStats: false,
+        calculateStats: true,
```
Comment (sedghi) [inline packages/tools/src/tools/annotation/PlanarFreehandROITool.ts:271]: can we set it back to false, it is brutal to calculate stats on each frame

File: packages/tools/src/tools/annotation/PlanarFreehandROITool.ts [hunk 2]
```diff
@@ -52,6 +52,31 @@ const { pointCanProjectOnLine } = polyline;
 const { EPSILON } = CONSTANTS;
 
 const PARALLEL_THRESHOLD = 1 - EPSILON;
+
+function calculatePerimeter(polyline, closed) {
```
Comment (sedghi) [inline packages/tools/src/tools/annotation/PlanarFreehandROITool.ts:56]: please refactor to utilities

File: packages/tools/src/tools/annotation/PlanarFreehandROITool.ts (hunk 1 above)
Comment (igoroctaviano) [reply packages/tools/src/tools/annotation/PlanarFreehandROITool.ts:271]: The stats were not being calculated / displayed. Maybe the problem is the tool implementation or when the stats are calculated?

**Review by IbrahimCSAE — APPROVED** [review]
works great thanks

**Pull request merged**